script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -921,7 +921,7 @@ I don't like what-ifs.
 -- Page 357, Offset 25169.
 -- どんな結果になるか分からないけど、とりあえずその時は、どこかに救いがあるような気がするでしょう？」
 You might not know where you're gonna end up,
- but in the moment you always feel like there's some kind of salvation out there, right?"
+ but in the moment, you always feel like there's some kind of salvation out there, right?"
```

Comment:
hmm, i think this one reads more naturally as a single, uninterrupted thought after the first clause. i say revert